services/discovery.go

```diff
@@ -65,8 +69,10 @@ func AddDiscoveredDevices(ctx context.Context, apiService openapi.DefaultAPIServ
 		if !exist {
 			newCredentials := *common.DefaultHostCredentials
 			newCredentials.IpAddress = hostDevice.Address
-			//Assign the actual ipAddress to customId to ensure its uniqueness
-			newCredentials.CustomId = newCredentials.CustomId + hostDevice.Address
+			//Assign the actual device name to customId
```

Comment:
* Please try to keep coding style consistent. Suggesting insert a space after the double slash
* It might be good to explain how .local tag is removed by using split here
* Please try to pack comment into a single line